include/swift/AST/Attr.def

```diff
@@ -272,6 +272,8 @@ DECL_ATTR(_implements, Implements,
           | NotSerialized | UserInaccessible,
           /* Not serialized */ 67)
 
+DECL_ATTR(NSKeyedArchiveLegacy, NSKeyedArchiveLegacy, OnClass, 68)
```

Comment:
I fretted over this. `NotSerialized` means that we won't necessarily see it in the generated interface for the module. While not wrong per se... it seems like useful information.